cg/meta/workflow/nf_analysis.py

```diff
@@ -278,5 +280,8 @@ def get_deliverables_for_case(self, case_id: str) -> PipelineDeliverables:
                 file[deliverable_field] = file[deliverable_field].replace(
                     "PATHTOCASE", str(self.get_case_path(case_id=case_id))
                 )
```

Comment:
I am not allowed to comment on all lines apparently but a minor thing - would it be possible to have inner for-loop, i.e. lines 276-282 in its own method with a name akin to `add_case_id_and_path_to_template`? Took me a while to figure out why that was necessary and why we do it in this order 😅 Feel free to disregard if out of scope, though.

Reply:
added `_update_template_with_case_information`